iphone/CoreApi/CoreApi.xcodeproj/project.pbxproj

```diff
@@ -607,6 +607,7 @@
 					"$(inherited)",
 					"$(OMIM_ROOT)/3party/glm",
 					"$(OMIM_ROOT)/3party/pugixml/pugixml/src",
+					"$(OMIM_ROOT)/3party/utf8cpp/include",
```

Comment:
Since utf8.h is included in base/strings_utils.hpp (the very base header), better to put it once into common.xcconfig

Instead of adding into almost each pbxproj

Reply:
Done